File: Fibonacci_nthTerm.c
#include <stdio.h>

// Recursive function to calculate Fibonacci numbers
int fibonacci(int n) {
    if (n <= 1)
        return n;
    return fibonacci(n - 1) + fibonacci(n - 2);
}

int main() {
    int n;
    printf("Enter the position of the Fibonacci term you want: ");
    scanf("%d", &n);
    
    // Print the nth Fibonacci number
    printf("Fibonacci term at position %d is: %d\n", n, fibonacci(n));
    
    return 0;
}

File: Fibonacci_nthTerm.py
# Recursive function to calculate Fibonacci numbers
def fibonacci(n):
    if n <= 1:
        return n
    return fibonacci(n - 1) + fibonacci(n - 2)

# Main code
n = int(input("Enter the position of the Fibonacci term you want: "))

# Print the nth Fibonacci number
print("Fibonacci term at position",n,"is",fibonacci(n))
# print(f"Fibonacci term at position {n} is: {fibonacci(n)}")